### ISSUE TITLE
PickType not working without @ApiProperty() decorator

### ISSUE THREAD

**Issue by FrizenVladyslav** [nestjs/swagger#674]
## Current behavior
PickType not working without @ApiProperty() decorator

## Input Code
```ts
import { PickType } from '@nestjs/swagger';
import { RegisterUserDto } from './register-user.dto';

export class ResetPasswordDto extends PickType(RegisterUserDto, [
  'email',
  'password',
]) {}
```

## Expected behavior
It should work with CLI plugin enabled

## Environment

<pre><code>
Nest version: 7.0.0
version @nestjs/swagger: 4.5.1
 
For Tooling issues:
- Node version: 12.16.1
- Platform:  Linux

**Comment by carlevans719** [comment]
I was having this error too & was busy putting together a repro when I noticed that I couldn't reproduce it in the new project. It turned out I had somehow managed to install version `0.12.0-rc.0` of `class-validator`. Downgrading to version `0.11.1` resolved the issue for me.

**Comment by kamilmysliwiec** [comment]
Fixed in 4.5.2
